pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -740,18 +742,9 @@ def _read_configuration():
               'deprecated and does nothing.')
         args.sdk_version = -1  # ensure it is not used
 
-    if args.permissions is None:
-        args.permissions = []
-    elif args.permissions:
-        if isinstance(args.permissions[0], list):
+    if args.permissions and isinstance(args.permissions[0], list):
             args.permissions = [p for perm in args.permissions for p in perm]
```

Comment:
Thanks for updating that PR so many times and also thanks for squashing your commits!
We have one more here:
`pythonforandroid/bootstraps/common/build/build.py:746:13: E117 over-indented`
Don't kill us now :grimacing: 
You can always run `tox` locally on your dev env to verify test and linting is OK if you don't want to wait for Travis to complain

Reply:
Sorry, is obviously my fault for not running pep8 locally ;)  I mostly rely on pycharm for catching formatting errors but it failed me at this one ;)